src/blueprints/tv2/timeline-object-factories/tv2-viz-timeline-object-factory.ts

```diff
@@ -130,7 +130,8 @@ export class Tv2VizTimelineObjectFactory implements Tv2GraphicsCommandTimelineOb
         noAutoPreloading: false,
         channelName: EngineName.FULLSCREEN,
         ...this.getFullGraphicOutTransitionProperties(blueprintConfiguration)
-      }
+      },
+      classes: ['full'] // Due to how Idents are ingested by Blueprints, if this is omitted VCPs will disappear when being taken from an Ident.
```

Comment:
I asked you while copying it over, if this value too should be moved other. You said no at that time. Turns out it was important :)

Happy to know the issue has been fixed.

Reply:
I remember. Do you also remember why I said to remove it?

Reply:
Yes somewhat. It was something in the lines of your last line in this PR's description.